src/helpers/search.js

```diff
@@ -48,10 +48,10 @@ export class SuggestionsHelper {
         if (a.part !== b.part) {
           return a.part - b.part;
         }
-        if (a.suggest.startsWith(query) && !b.suggest.startsWith(query)) {
+        if (a.suggest.toLowerCase().startsWith(query) && !b.suggest.toLowerCase().startsWith(query)) {
```

Comment:
toLowerCase is potentially called for multiple times for every suggestion.
Probably it's better to map them to lower case before sort()?

Reply:
changed